src/Microsoft.Health.Dicom.Api/Registration/DicomServerApplicationBuilderExtensions.cs

```diff
@@ -56,6 +72,39 @@ public static IApplicationBuilder UseDicomServer(this IApplicationBuilder app)
             app.UseStaticFiles();
         }
 
+        app.UseRouting();
+        useDevelopmentIdentityProvider?.Invoke(app);
```

Comment:
I am wondering if we can rewrite this function to not require such verbose arguments. Perhaps we don't even need this method. Like, we could create an extension method for the OHIF viewer addition. Otherwise, everything else could be explicitly added by a web project in the `Program.cs` or `Startup.cs`, right?

Reply:
Moved these to `startup.cs` as per the discussion offline.